crates/rolldown/tests/esbuild/ts/export_type_issue379/artifacts.snap

```diff
@@ -46,11 +20,15 @@ var b_exports = {};
 __export(b_exports, { foo: () => foo$2 });
 let foo$2 = 123;
 
+//#endregion
+//#region test.ts
+var Test = void 0;
```

Comment:
Negative situation for `shimMissingExports` vs native support for detection on `export type`.